package.json

```
@@ -77,5 +77,8 @@
   "peerDependencies": {
     "react": "> 15.0.0",
     "react-native": "> 0.50.0"
+  },
+  "dependencies": {
+    "uuid": "^8.3.0"
```

Can we fix the version?

```suggestion
    "uuid": "8.3.0"
```